### ISSUE TITLE
Raspberry PI Zero 2W Bookworm

### ISSUE THREAD

**Issue by magni-optitog** [raspberrypi/libcamera#162]
Does libcamera support Raspberry PI Zero 2W running Bookworm? I have followed the conventional guides and cannot get the camera to appear when running `libcamera-vid --list-cameras`. Any advice is welcome.

**Comment by naushir** [comment]
Yes it ought to work fine.  Please follow the instruction here:
https://forums.raspberrypi.com/viewtopic.php?t=362707

**Comment by magni-optitog** [comment]
I followed `How to check if an official Raspberry Pi camera is working` and the raspberry pi still outputs that no camera is detected. The same is true of `rpicam-hello`, I am using an OV2311 camera https://docs.arducam.com/Raspberry-Pi-Camera/Pivariety-Camera/2MP-OV2311/. I have no idea where to look for inspiration although I have tried many things. The last step I took was to do a clean installation and see if the camera is detected, to no avail.
